packages/lesswrong/server/users/cron.ts

```diff
@@ -0,0 +1,35 @@
+import { addCronJob } from '../cronUtil';
+import Users from "../../lib/collections/users/collection";
+import { ModeratorActions } from "../../lib/collections/moderatorActions";
+import { allRateLimits } from "../../lib/collections/moderatorActions/schema";
+import { getSignatureWithNote } from "../../lib/collections/users/helpers";
+import * as _ from 'underscore';
+import moment from 'moment';
+
+
+addCronJob({
+  name: 'expiredRateLimitsReturnToReviewQueue',
+  interval: 'every 24 hours',
+  async job() {
+  
+  
+    const startOfDay = moment(new Date()).subtract(1, 'days').toDate()
+    const endOfDay = new Date() 
+    
+    const rateLimitsExpiringToday = await ModeratorActions.find({type: {$in: allRateLimits}, endedAt: {$gte: startOfDay, $lt: endOfDay}}).fetch();
```

Comment:
I think dropping the `$lt: endOfDay` thing would make this more robust. If this happens to not run for some reason or a moderator action gets inserted with an endDate in the past somehow I'm assuming you would still want it to expire

Reply:
Not sure if I misunderstood you or the opposite. This doesn't cause them to expire, it causes them to show up in the review queue again. So if you ran it for all ratelimits that were in the past, then they'd all keep getting add to the review queue again.

Reply:
(also I think you'd mean `startOfDay` too)